.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -37,3 +37,8 @@ To be verified by following test cases:
 - [ ] 1. Pass/Fail (SW version, date, verifier)
 - [ ] 2. Pass/Fail (SW version, date, verifier)
 
+## Traceability
+
+_instruction: Check that the pull request is linked to an issue, if not then link it_
```

Comment:
Shouldn't the instruction be the other way around, asking to check that the issue is linked to at least one pull request?